cmd/kar-controllers/app/options/options.go

```diff
@@ -30,6 +30,10 @@ type ServerOption struct {
 	Dispatcher	bool
 	AgentConfigs 	string
 	SecurePort	int
+	DynamicPriority	bool  // If DynamicPriority=true then no preemption is allowed by program logic
+	Preemption 		bool  // Preemption is not allowed under DynamicPriority
+	SendUpdate		bool  // Send QueueJobStatus Updatae() to Etcd.  Not essential for Controller functions
+	Demo	bool  // Demo: add delay after moving HeadOfLine job to eventQueue, before examine qjqueue again
```

Comment:
Please format the types 'bool' to align with types above :D

Reply:
Corrected.

Reply:
Alignment for `DynamicPriority` looks great! Can you do the same for `Preemption` and `SendUpdate`?  Thanks so much!

Reply:
https://github.com/zhangliresearch/multi-cluster-app-dispatcher/blob/config-for-dynamic-priority/cmd/kar-controllers/app/options/options.go#L33 to line36 already show good alignment.

Reply:
Ah, sorry, please push any updates to your origin branch and it will update the PR as well and the changes will be reflected.

Reply:
Latest updates to PR reflected good alignmet.  Thank you!